pages/en/glossary.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Glossary
+---
+
+- **The Graph**: A decentralized protocol for indexing and querying data.
+
+- **Query**: A request for data. In the case of The Graph, a query is a request for data from a subgraph that will be answered by an Indexer.
+
+- **GraphQL**: A query language for APIs and a runtime for fulfilling those queries with your existing data. The Graph uses GraphQL to query subgraphs.
+
+- **Subgraph**: A custom API built on blockchain data that can be queried using [GraphQL](https://graphql.org/). Developers can build, deploy and publish subgraphs to The Graph's decentralized network. Then, Indexers can begin indexing subgraphs to make them available to be queried by subgraph consumers.
+
+- **Hosted Service**: A temporary scaffold service for building and querying subgraphs while The Graph's decentralized network matures its cost of service, quality of service, and developer experience.
```

Comment:
maybe "as The Graph's decentralized network is maturing.."
- On that note, isn't the Quality of Service better on decentralized network than on hosted? (I think we say so in the docs)

Reply:
I don't think we say it's better anywhere public yet. Closest we come is [here](https://thegraph.com/docs/en/network-transition-faq/#how-does-quality-of-service-currently-compare-between-the-hosted-service-and-the-decentralized-network), but it's not necessarily saying the QoS is better, but mostly comparable.